app/assets/stylesheets/application.scss

```diff
@@ -50,3 +50,38 @@
 #search-bar {
   margin-right: 30px;
 }
+
+.profile-body {
+  margin: 0 auto;
+  width: 80%;
+  display: block;
+}
+
+.profile-body img {
+  width: 150px;
+  height: auto;
+  float: left;
+}
+
+.profile-text {
+  display: block;
+  margin: 0 auto;
+  float: none;
+}
+
+.profile-text ul {
+  display: inline-block;
+}
+
+.profile-text li {
+  list-style: none;
+}
+
+.profile-text li span {
```

Comment:
Merge rule `.profile-text li span` with rule on line 66<br>Selector should have depth of applicability no greater than 2, but was 3